tests/storage/xlease_test.py

```diff
@@ -151,13 +150,12 @@ def close(self):
 ])
 def tmp_vol(request):
     storage, alignment = request.param
-    if not storage.exists():
-        pytest.xfail("{} storage not available".format(storage.name))
-    backend = xlease.DirectFile(storage.path)
-    tv = TemporaryVolume(backend, alignment, block_size=storage.sector_size)
-
-    yield tv
-    tv.close()
+    with Backend(storage):
+        backend = xlease.DirectFile(storage.path)
+        tv = TemporaryVolume(
+            backend, alignment, block_size=storage.sector_size)
+        yield tv
+        tv.close()
```

Comment:
Looks like good way to extend the backend.

Reply:
This is very specific for xlease testing, no?

Reply:
yes, it shows we can extend the backend easily.